ee/clickhouse/queries/funnels/funnel.py

```diff
@@ -66,7 +67,10 @@ def _format_single_funnel(self, result, with_breakdown=False):
                 serialized_result.update({"average_conversion_time": None})
 
             if with_breakdown:
-                serialized_result.update({"breakdown": result[-1][1:-1]})  # strip quotes
+                resulting_breakdown = (
+                    result[-1][1:-1] if isinstance(result[-1], str) else Cohort.objects.get(pk=result[-1]).name
```

Comment:
Once a decision on #5043 is made, this will probably have to change.
```suggestion
                    result[-1] if isinstance(result[-1], str) else Cohort.objects.get(pk=result[-1]).name
```